src/test/groovy/org/jenkinsci/plugins/globalEventsPlugin/acceptance/StepDefs.groovy

```diff
@@ -5,12 +5,7 @@ import cucumber.api.java.en.Given
 import cucumber.api.java.en.Then
 import cucumber.api.java.en.When
 import hudson.util.FormValidation
-import org.jenkinsci.plugins.globalEventsPlugin.GlobalEventsPlugin
-import org.jenkinsci.plugins.globalEventsPlugin.GlobalEventsPluginTest
-import org.jenkinsci.plugins.globalEventsPlugin.GlobalRunListener
-import org.jenkinsci.plugins.globalEventsPlugin.GlobalComputerListener
-import org.jenkinsci.plugins.globalEventsPlugin.GlobalQueueListener
-import org.jenkinsci.plugins.globalEventsPlugin.LoggerTrap
+import org.jenkinsci.plugins.globalEventsPlugin.*
```

Comment:
I didn't get `import *` policy. In some places you changed `*` to set of imports, here you changes several imports to one asterisk :)